apps/allocations/app/components/App/App.js

```diff
@@ -32,7 +35,8 @@ const App = () => {
         onResolve={handleResolveLocalIdentity}
         onShowLocalIdentityModal={handleShowLocalIdentityModal}
       >
-        <Overview />
+        {path === '/' && <Overview />}
```

Comment:
```suggestion
        {!path.match(BUDGETS_REGEX) && <Overview />}
```
Until we implement 404 pages, we should at least fail gracefully into the Overview page when a user submits an incorrect url. Right now if I type '/overview' into the url (or anything else that does not match the regex), I get a blank page.

Reply:
In a lower comment I offered up another solution to this issue, so you can ignore this.

Reply:
I'm going to go with something closer to this solution, I think 😄 

Your second suggestion seems very specific to the current routing structure of Allocations, and I want to try to come up with patterns that will be useful in more apps and as we add new routes to Allocations.